stubparser/src/org/checkerframework/stubparser/ASTParser.java

```diff
@@ -1,21 +1,21 @@
 /* Generated By:JavaCC: Do not edit this line. ASTParser.java */
-/*
- * Copyright (C) 2008 Júlio Vilmar Gesser.
- * 
- * This file is part of Java 1.5 parser and Abstract Syntax Tree.
- *
- * Java 1.5 parser and Abstract Syntax Tree is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Lesser General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * Java 1.5 parser and Abstract Syntax Tree is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * along with Java 1.5 parser and Abstract Syntax Tree.  If not, see <http://www.gnu.org/licenses/>.
+/*
```

Comment:
Aren't most of the changes in this class whitespace changes? I don't see a difference. Can you remove these again?